packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/seedlessExportPhrase/refreshSeedlessToken/verifyTotpCode.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react'
+import { VerifyCode } from 'features/onboarding/components/VerifyCode'
+import { TotpErrors } from 'seedless/errors'
+import { Result } from 'types/result'
+import { useSeedlessMnemonicExportContext } from 'features/accountSettings/context/SeedlessMnemonicExportProvider'
+import { useNavigation, useRouter } from 'expo-router'
```

Comment:
/**
 * Temporarily import "useNavigation" from @react-navigation/native.
 * This is a workaround due to a render bug in the expo-router version.
 * See: https://github.com/expo/expo/issues/35383
 * TODO: Adjust import back to expo-router once the bug is resolved.
 */